edit/namespace/index.vue

```diff
@@ -24,7 +24,7 @@ export default {
     let originalQuotaId = null;
 
     if (!!this.originalValue) {
-      originalQuotaId = `${ this.originalValue.metadata.name }/default-quota`;
+      originalQuotaId = `${ this.originalValue?.metadata?.name }/default-quota`;
```

Comment:
I assume this was breaking something, but how?

`originalModel` (->`originalValue`) is supposed be null on create, and on edit the name can't be not-set?

Reply:
originalModal is being created regardless of mode:
https://github.com/mantis-toboggan-md/dashboard/blob/masthead/components/ResourceDetail/index.vue#L88

I can remove it, but I'm not sure why it's being created in the first place

Reply:
..Apparently I've changed it in my copy but haven't pushed it yet.

```javascript
    originalModel = null;
    model = await store.dispatch('cluster/create', data);
```